docs/docs/user_guide/user_library.md

```diff
@@ -0,0 +1,67 @@
+
+To understand more about definitions and their role in AaC, see the [Definitions](./aac_language/#definitions) section of the AaC Modeling Language Reference page.
+
+While AaC plugins aren't limited to only contributing definitions, this is the only plugin functionality necessary to create a distributable user library for the AaC tool. In order to support user libraries, plugins include their definitions as part of their registration in AaC. For references on how to register an AaC plugin, see [Plugins](../../old/Plugins) and [Plugin Developer Guide](../dev_guide/plugin_dev_guide)
+
+## Inactive User Library Plugin
+When a plugin has definitions associated with it, but the plugin has not been activated, the definitions are not available for use by AaC. To confirm a plugin has been installed, but not registered, run the following command: `aac --inactive`.
+
+![Inactive Plugins](../../../assets/images/user_library/inactive_plugins.png)
+## Active User Library Plugin
+In order to use the definitions in a user library plugin, you must make sure that it is actively contributing definitions to the context.
+
+You can activate a registered plugin by using the following command: `aac activate-plugin <plugin name>`.
+
+![Activate Command](../../../assets/images/user_library/activate_command.png)
+
+Verify the plugin has been activated by running the following command: `aac --active`
+
+![List Active](../../../assets/images/user_library/list_active.png)
+
+Once you have verified that your plugin has been activated, you are able begin referencing the definitions in your plugin. If you get validation errors about undefined types, double check that your plugin is listed as actively installed.
+
+To demonstrate this behavior, take this example aac file:
+```yaml
+schema:
+    name: ProtobufTypeExample
+    fields:
+        - name: int64Field
+          type: int64
+          description: This field references a primitive type provided by Gen-Protobuf
+```
+
+It references a type `int64` provided by the plugin `gen-protobuf`. If we validate this file with the plugin active, it'll successfully validate.
+![Validate example](../../../assets/images/user_library/validate_example.png)
+
+If we deactivate `gen-protobuf` and run the validation again, we'll see an error indicating our plugin type isn't defined.
+
+![Revalidate example](../../../assets/images/user_library/revalidate_example.png)
+
+Once you have verified that your plugin has been activated, you are able to check what plugin definitions, plugin commands, and such are available by running these commands: `aac <plugin name> --definitions` *think this is the correct command, please correct if not*
+
+`<include screen shot of list results of plugin definitions>`
+
+`aac <plugin name> --commands` *think this is the correct command, please correct if not*
+
+`<include screen shot of list results of plugin commands>`
```

Comment:
We don't currently have plugin commands that will do this, but it does seems like an obvious one. I've created draft stories for these features for the future. 
#610 
#611 

```suggestion
```

Reply:
I removed it and pushed the updated.